docs/topics/db/queries.txt

```diff
@@ -995,16 +974,9 @@ example::
     >>> Dog.objects.filter(data__contained_by={'breed': 'collie'})
     <QuerySet [<Dog: Fred>]>
 
-.. note::
-
-    On PostgreSQL, the implementation uses the SQL operator ``<@``. On MySQL
-    and MariaDB, it uses the SQL function ``JSON_CONTAINS`` like ``contains``
-    lookup but with the arguments switched.
-
-.. admonition:: Oracle and SQLite users
+.. admonition:: Oracle and SQLite
 
-    Due to the limitation on Oracle and SQLite database backends, this lookup
-    is not supported.
+    This lookup is not supported on Oracle and SQLite.
```

Comment:
I forgot to edit this, it's actually supported with the help of custom functions.

Reply:
> I forgot to edit this, it's actually supported with the help of custom functions.

TBH, we don't have time to play with support on Oracle before the feature freeze.

Reply:
Oh, I forgot to specify. I mean it's supported on SQLite 😅